ViteMaDose/Views/Credits/Header/CreditSectionView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CreditSectionView.swift
```

Comment:
Are you sure _CreditSectionView_ is used? I don't think so ; these objects should be deleted.